unlock-app/src/middlewares/storageMiddleware.js

```diff
@@ -140,7 +140,8 @@ const storageMiddleware = config => {
           const { emailAddress, password } = action
           storageService
             .getUserPrivateKey(emailAddress)
-            .then(key => {
+            .then(result => {
+              const key = result.data.passwordEncryptedPrivateKey
```

Comment:
`storageService` should actually handle the logic of extracttibg the  `data` from `result` and only yield `passwordEncryptedPrivateKey`. Could you change that?